src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -375,10 +411,31 @@ private IReachHttpResponse Send(HomeWorkspaceModel workspace, WorkspacePropertie
                     workspaceProperties);
         }
 
+        private Task<IEnumerable<Workspace>> GetWorkspaces(string name)
+        {
+            if (reachClient == null)
+            {
+                reachClient = new WorkspaceStorageClient(authenticationProvider, serverUrl);
+            }
```

Comment:
I don't really see why we're lazily init'ing this type. Why not just make it `private readonly`?